src/PostalCodes/PostalCode.cs

```diff
@@ -158,6 +189,16 @@ public virtual int CompareTo(PostalCode other)
             return other == null ? 1 : PostalCodeStringComparer.Default.Compare(PostalCodeString, other.PostalCodeString);
         }
 
+        /// <summary>
+        /// Compares the current object with another object of the same type.
+        /// </summary>
+        /// <param name="other">An object to compare with this object.</param>
+        /// <returns>A value that indicates the relative order of the objects being compared. The return value has the following meanings: Value Meaning Less than zero This object is less than the <paramref name="other" /> parameter.Zero This object is equal to <paramref name="other" />. Greater than zero This object is greater than <paramref name="other" />.</returns>
+        public virtual int CompareTo(string other)
```

Comment:
Added some overloads for the comparison functions to make it easier to compare to the lowest- and highest-expanded postal code strings in `PostalCodeRange.Contains` without needing to create `PostalCode` objects or duplicate any string comparison logic in `PostalCodeRange`.

Reply:
Would it be possible a GB postal code (PostalCode class) to be compared with a string?

Reply:
When `PostalCodeRange.Contains(PostalCodeRange, PostalCode)` is called, it validates that the specified postal code is [valid for comparison](https://github.com/Cimpress-MCP/PostalCodes.Net/blob/master/src/PostalCodes/PostalCodeRange.cs#L355) with the start and end postal codes of the range before any comparison happens. [For GB postal codes](https://github.com/Cimpress-MCP/PostalCodes.Net/blob/master/src/PostalCodes/Extensions/GBPostalCode.cs#L9), this involves verifying that the postal codes being compared have the same format.

If those checks pass, then the given postal code should be valid for comparison with the start and end of the range, so the lowest- and highest-expanded strings will be used for comparison.